gcp/extract/tests/test_gcmfilter.py

```diff
@@ -8,32 +7,36 @@ def test_included_models():
     config = {
         "drop-models": 'access1-0'
     }
-    configs.included_models(config) == (None, ['access1-0'])
+    assert configs.included_models(config) == (None, ['access1-0'])
 
     config = {
         "only-models": ['ccsm4', 'access1-0']
     }
-    configs.included_models(config) == (['ccsm4', 'access1-0'], [])
+    assert configs.included_models(config) == (['ccsm4', 'access1-0'], [])
 
     config = {
         "only-models": ['ccsm4', 'access1-0'],
         "drop-models": 'access1-0'
     }
-    configs.included_models(config) == (['ccsm4'], [])
+    assert configs.included_models(config) == (['ccsm4'], [])
 
 def test_march2018_filepath():
     config = {
         "drop-models": 'access1-0'
     }
-    weights.march2018_filepath('rcp85', config, True) == '/shares/gcp/climate/BCSD/SMME/SMME-weights/rcp85_SMME_weights_no_access1-0.tsv'
+    assert weights.march2018_filepath('rcp85', config, True) == '/shares/gcp/climate/BCSD/SMME/SMME-weights/rcp85_SMME_weights_no_access1-0.tsv'
 
     config = {
         "only-models": ['ccsm4', 'access1-0']
     }
-    configs.march2018_filepath('rcp85', config, True) == '/shares/gcp/climate/BCSD/SMME/SMME-weights/rcp85_SMME_weights_of_access1-0_ccsm4.tsv'
+    assert weights.march2018_filepath('rcp85', config, True) == '/shares/gcp/climate/BCSD/SMME/SMME-weights/rcp85_SMME_weights_of_access1-0_ccsm4.tsv'
 
     config = {
         "only-models": ['ccsm4', 'access1-0'],
         "drop-models": 'access1-0'
     }
-    configs.march2018_filepath('rcp85', config, True) == '/shares/gcp/climate/BCSD/SMME/SMME-weights/rcp85_SMME_weights_of_access1-0.tsv'
+    assert weights.march2018_filepath('rcp85', config, True) == '/shares/gcp/climate/BCSD/SMME/SMME-weights/rcp85_SMME_weights_of_access1-0.tsv'
```

Comment:
@jrising I am getting an `AssertionError` here. Isn't the expected string in fact `...of_ccsm4.tsv` here? If I understand correctly `configs.included_models()`.

Reply:
Oops! I missed that. Fixed now.